nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -665,6 +668,7 @@ def filter_func(point: StatisticPoint) -> bool:
                 transformation_layout.register(command)
 
         quantized_model = model_transformer.transform(transformation_layout)
+        quantized_model = self._apply_device_pass(self._target_device, quantized_model)
```

Comment:
What is the benefit of using a post-processing pass? I mean, If you will make it as step before insertion fake quantize it allows us to avoid double model transformation.

Reply:
Changed.